### DISCUSSION TITLE
Escritório da APyB em Caxias

### DISCUSSION THREAD

**Discussion by meu-bem** [apyb/comunidade#132] [Geral]
Oi pessoal. Tudo bem?
Sabem como funciona o endereço físico da APyB no RS? Fica alguém lá? É que a empresa que fez os nossos copos cometeu o equívoco de mandar para lá e informou que para retornar seria menos burocrático se alguém lá recusasse a entrega alegando não ser o endereço correto.

**Comment by meu-bem** [comment]
Aparentemente não há pessoa viável que possa cumprir esse papel lá.

**Comment by meu-bem** [comment]
Link pro rastreio: https://radar.tntbrasil.com.br/radar/public/localizacaoSimplificada.do
Tipo de cliente: Destinatário
CNPJ: 09344485000105
Filial de origem: Todos
Tipo de documento: Nota Fiscal
Número do documento: 5914

**Reply by naanadr** [reply]
@MateusSantosMeuBem vocês chegaram a entrar em contato com a transportadora para ver se seria possível cancelar o envio do produto?

**Reply by meu-bem** [reply]
@naanadr, quem tratou com a transportadora foi o fornecedor. Ele informou que inicialmente que se o pacote não fosse recebido, ele retornaria e eles encaminhariam para para o endereço correto. Mas que seria menos burocrático:
1. Receber e eles mandarem outra remessa buscar;
2. Alguém lá informar que aquele não era o endereço correto pra voltar o mais breve.

**Reply by cecivieira** [reply]
O pacote saiu para entrega nesta manhã. 
@MateusSantosMeuBem não sabemos como será procedido o recebimento ou o não recebimento dessa mercadoria.
Sugiro você acompanhar junto com a transportadora. Caso o pacote tenha sido recebido, nós verificaremos com quem está na residencia atualmente.

**Reply by meu-bem** [reply]
@cecivieira, farei isso. Obrigado!
